test/indexer/__init__.py

```diff
@@ -186,8 +186,18 @@ def verify_sorted_lists(data: AnyJSON, path: FieldPath = ()) -> int:
                                    for val in cast(JSONs, data)
                                    for k, v in val.items())
                     elif isinstance(data[0], (type(None), bool, int, float, str)):
-                        self.assertEqual(data, sorted(data, key=lambda x: (x is None, x)))
-                        return 1
+                        # FIXME: Field types don't express ordering requirements
+                        #        https://github.com/DataBiosphere/azul/issues/4664
+                        # Reverse entity index and entity field index; pertinent to path
+                        ei, efi = (-2, -1)
+                        if path[ei] == 'projects' and path[efi] in ('laboratory',
```

Comment:
Are these variables really necessary? I think it's just as clear to write 
```suggestion
                        if path[-2] == 'projects' and path[-1] in ('laboratory',
```
given the context. Especially since I had to read a comment to understand what the variables' names were supposed to mean.

Reply:
Yeah, the comment above was describing the reason behind ei, and efi, I thought it would add clarity 😅.